libs/labelbox/src/labelbox/schema/send_to_annotate_params.py

```diff
@@ -29,14 +31,23 @@ class SendToAnnotateFromCatalogParams(TypedDict):
         ConflictResolutionStrategy.KEEP_EXISTING.
     :param batch_priority: Optional[int] - The priority of the batch. Defaults to 5.
     """
-
-    source_model_run_id: Optional[str]
-    predictions_ontology_mapping: Optional[Dict[str, str]]
-    source_project_id: Optional[str]
-    annotations_ontology_mapping: Optional[Dict[str, str]]
-    exclude_data_rows_in_project: Optional[bool]
-    override_existing_annotations_rule: Optional[ConflictResolutionStrategy]
-    batch_priority: Optional[int]
+    model_config = ConfigDict(extra='forbid')
```

Comment:
we generally do not enforce this, please remove

Reply:
I can remove it but the only reason I added was because this was a unique parameter in our SDK. We pass in a dictionary of parameters instead of just the arguments which I think we do everywhere else only these methods are like this. From getting on a call with a customer one thing I saw accidentally typos with the hard strings like if I put source_project_ids as a key instead of source_project_id and they get confused why nothing happened since it wont error out and everything has a default value.

Reply:
I removed it!

Reply:
Changed!

Reply:
wait your argument made sense to me... I am good either way